docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Manage your Applications using Kyma and Compass Console UI
+type: Tutorial
+---
+
+This tutorial presents an end-to-end use case scenario that shows how to connect an external Application to Compass so that its lambda calls and gets the order services. While going through this tutorial, you will navigate between two UI views:
```

Comment:
This tutorial presents basic flow where user registers manually external Application APIs into Compass and then expose them into Kyma Runtime. Next, in the Kyma Runtime users creates lambda which calls order service API. While going through this tutorial, you will navigate between two UI views: ...

Comment: Next tutorials will show automatic connection between external Application and Compass. That's a reason I've changed your tutorial description.